stacks-signer/src/signerdb.rs

```diff
@@ -545,9 +559,14 @@ static SCHEMA_7: &[&str] = &[
     "INSERT OR REPLACE INTO db_config (version) VALUES (7);",
 ];
 
+static SCHEMA_8: &[&str] = &[
+    CREATE_INDEXES_8,
+    "INSERT OR REPLACE INTO db_config (version) VALUES (8);",
```

Comment:
I realized when looking over the DBs, that this still doesn't do what we intended, to only have one entry in the `db_config` table.

Reply:
Yeah, this would work instead...

```sql
UPDATE db_config SET version = 8;
```

Reply:
Can we be sure that there is already 1 row in the table? I think it looks like when no table exists, we go through all of the migrations, so that would indeed ensure we have at least one entry. We could probably simplify that case though.

Reply:
Yeah, if we're executing this line, there has to be at least one row (because we got that version = 7).

I'd note that, in a prod DB, we have two rows - one for versions 5 and 6.

Reply:
So what's the decision here? Update to `UPDATE` and just ignore the fact that we'll have two 8s, or clean up the DB again so that we just end up with one 8, and make sure we do it right for future versions?

Reply:
Oh, I might just prefer letting the logic add another row /shrug

Reply:
yeah, that's fine too. it's not like there will be thousands. in that case, let's just make this an `INSERT`.